app/views/overrides/http_proxies/_update_setting_input.html.erb

```diff
@@ -0,0 +1,16 @@
+<% if @http_proxy.new_record? %>
+  <div class="clearfix">
+    <div class="form-group ">
+      <label class="col-md-2 control-label" for="http_proxy_default_content">
+        Default Content HTTP proxy
+      </label>
+      <div class="col-md-4">
+        <%= check_box_tag 'http_proxy[default_content]',
+                          '1',
+                          params.dig('http_proxy', 'default_content') == '1',
+                          id: 'http_proxy_default_content' %>
+        Set this proxy as the default for content, updating the 'Default HTTP Proxy' setting.
+      </div>
```

Comment:
When we create a http proxy with default_content_proxy set to true with UI, it gets created successfully and setting is updated, but when we click on created proxy on edit page we don't see the value set for default_content_proxy, so I feel it would be better to include it on edit page too

Reply:
And, If we decide to add it to the edit page, will it be possible to change it or will it remain grayed?

Reply:
It's tied to a Setting, so if it were included on the edit page it would have to be disabled. It would just be an indicator of whether this current HTTP proxy is set as the default.